flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/StormTuple.java

```diff
@@ -44,6 +45,21 @@
 	/** The schema (ie, ordered field names) of the tuple */
 	private final Fields schema;
 
+	private final int taskId;
+	private final String streamId;
+	private final MessageId id;
+	private final String componentId;
+
```

Comment:
There is a JIRA for this. https://issues.apache.org/jira/browse/FLINK-2721 I am already working on it. I think that FLINK-2721 should be resolved before this.